scripts/scil_compute_freewater.py

```diff
@@ -47,11 +48,11 @@ def _build_arg_parser():
     p.add_argument('in_bvec',
                    help='b-vectors filename, in FSL format (.bvec).')
 
-    p.add_argument('--in_mask',
+    p.add_argument('--mask',
                    help='Brain mask filename.')
     p.add_argument('--out_dir', default="results",
                    help='Output directory for the Free Water results. '
-                        '[current_directory]')
+                        '[%(default)s]')
```

Comment:
You told me by chat that if the output directory is `./` the script will remove everything in the input dir, which of course is bad. Could you add a check to make sure output_dir is not the current directory?